Fix some more names in the doc (#4)
valkey-io/valkey-doc: commands/info.md

## Notes

Please note depending on the version of Valkey some of the fields have been
added or removed. A robust client application should therefore parse the
result of this command by skipping unknown properties, and gracefully handle
missing fields.

> Here is the description of fields for Valkey >= 2.4.

Reviewer: ```suggestion
Here is the description of fields.
```

Here is the meaning of all fields in the **server** section:

> *   `redis_version`: Version of the Valkey server

Reviewer: ```suggestion
*   `redis_version`: Redis version this Valkey server is compatible with
```

*   `blocked_clients`: Number of clients pending on a blocking call (`BLPOP`,
     `BRPOP`, `BRPOPLPUSH`, `BLMOVE`, `BZPOPMIN`, `BZPOPMAX`)
*   `tracking_clients`: Number of clients being tracked (`CLIENT TRACKING`)

> *   `pubsub_clients`: Number of clients in pubsub mode (`SUBSCRIBE`, `PSUBSCRIBE`, `SSUBSCRIBE`). Added in Valkey 8.0

Reviewer: Let's just remember we have mentioned this future version in the docs....

*   `connected_slaves`: Number of connected replicas

> If the server is configured with the `min-slaves-to-write` (or starting with Valkey 5 with the `min-replicas-to-write`) directive, an additional field is provided:

Reviewer: ```suggestion
If the server is configured with the `min-replicas-to-write` directive, an additional field is provided:
```

[hcgcpgp]: http://code.google.com/p/google-perftools/

> **A note about the word slave used in this man page**: Starting with Valkey 5, if not for backward compatibility, the Valkey project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.

Reviewer: ```suggestion
**A note about the word slave used in this man page**: If not for backward compatibility, the Valkey project does not use the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.
```

> **Modules generated sections**: Starting with Valkey 6, modules can inject their info into the `INFO` command, these are excluded by default even when the `all` argument is provided (it will include a list of loaded modules but not their generated info fields). To get these you must use either the `modules` argument or `everything`.,

Reviewer: ```suggestion
**Modules generated sections**: Modules can inject their info into the `INFO` command. These are excluded by default even when the `all` argument is provided. (It will include a list of loaded modules but not their generated info fields). To get these you must use either the `modules` argument or `everything`.
```
